### PR TITLE
Shipping/docker metrics

### PR DESCRIPTION
**Pull request by shalper**
# What changed

----

## Pages to review

- https://deploy-preview-522--logz-docs.netlify.app/shipping/metrics-sources/docker.html

### CODE REVIEW

File: _source/logzio_collections/_log-sources/python.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ you need to set every argument that comes before it.
 | timeout <span class="default-param">`3`</span> | Time to wait between log draining attempts, in seconds. |
 | listener-url <span class="default-param">`https://listener.logz.io:8071`</span> | Listener URL and port. <br> {% include log-shipping/replace-vars.html listener=true %} |
 | debug-flag <span class="default-param">`False`</span> | Debug flag. To print debug messages to stdout, `True`. Otherwise, `False`. |
-|backup-logs <span class="default-param">`True`</span>| If set to False, disables the local backup of logs in case of failure. |
+| backup-logs <span class="default-param">`True`</span>| If set to False, disables the local backup of logs in case of failure. |
```
Comment (mirii1994) [inline _source/logzio_collections/_log-sources/python.md:129]: wasn't this part of the python-handler PR? I don't think it should be in here...

File: _source/logzio_collections/_metrics-sources/docker.md [hunk 2]
```diff
@@ -80,6 +81,7 @@ logzio/docker-collector-metrics
 | LOGZIO_TYPE <span class="default-param">`docker-collector-metrics`</span> | This field is needed only if you're shipping metrics to Kibana and you want to override the default value. <br> In Kibana, this is shown in the `type` field. Logz.io applies parsing based on `type`. |
 | LOGZIO_LOG_LEVEL <span class="default-param">`"INFO"`</span> | The log level the module startup scripts will generate. |
 | LOGZIO_EXTRA_DIMENSIONS | Semicolon-separated list of dimensions to be included with your metrics (formatted as `dimensionName1=value1;dimensionName2=value2`). <br> To use an environment variable as a value, format as `dimensionName=$ENV_VAR_NAME`. Environment variables must be the only value in the field. If an environment variable can't be resolved, the field is omitted. |
+| DEBUG <span class="default-param">`"false"`</span> | Set to `true` if you want Metricbeat to run in debug mode.<br/> **Note:** Debug mode tends to generate a lot of debugging output, so you should probably enable it temporarily only when an error occurs while running the docker-collector in production.  |
```
Comment (mirii1994) [inline _source/logzio_collections/_metrics-sources/docker.md:84]: Awesome! this line should be added not only to to the docker module but for the system metrics & aws metrics, too.

File: _source/logzio_collections/_log-sources/python.md (hunk 1 above)
Comment (shalper) [reply _source/logzio_collections/_log-sources/python.md:129]: it was a formatting correction. There was a missing space.

File: _source/logzio_collections/_metrics-sources/docker.md (hunk 2 above)
Comment (shalper) [reply _source/logzio_collections/_metrics-sources/docker.md:84]: added :)

**Comment by shalper** [comment]
> Maybe we can give an example on how to add this namespace to the command: 'AWS/EC2,CWAgent' instead of just saying to specify the namespace.

@yotamloe Can you provide the command for this example?

**Pull request merged**